packages/libssh.rb

```diff
@@ -3,34 +3,26 @@
 class Libssh < Package
   description 'libssh is a multiplatform C library implementing the SSHv2 and SSHv1 protocol on client and server side.'
   homepage 'https://www.libssh.org/'
-  version '0.8.1'
+  version '0.9.5'
   compatibility 'all'
-  source_url 'https://www.libssh.org/files/0.8/libssh-0.8.1.tar.xz'
-  source_sha256 'd17f1267b4a5e46c0fbe66d39a3e702b8cefe788928f2eb6e339a18bb00b1924'
-
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libssh-0.8.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libssh-0.8.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libssh-0.8.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libssh-0.8.1-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '0928c31849b027b3188b765c4ee7434cad046523cf4a54355e125421baaf64c5',
-     armv7l: '0928c31849b027b3188b765c4ee7434cad046523cf4a54355e125421baaf64c5',
-       i686: 'ee40bfa70d6b0fd644ab58f081dd3c82f29ca0f28586698eed55c79d79656c47',
-     x86_64: '8cca9b0403091a838e65b2306cb5bb3b8ed549502d7e54c442af0fac1a3fc671',
-  })
+  source_url 'https://www.libssh.org/files/0.9/libssh-0.9.5.tar.xz'
+  source_sha256 'acffef2da98e761fc1fd9c4fddde0f3af60ab44c4f5af05cd1b2d60a3fa08718'
 
   depends_on 'libgcrypt'
-
+    
+  def self.patch
+      `echo "$(sed '/\# global needed variable/a set\(CMAKE_C_FLAGS \"-fstack-protector-strong\"\)~\
+add_compile_options\(-fstack-protector-strong\)~\
+link_libraries\(-fstack-protector-strong\)' CMakeLists.txt)" > ./CMakeLists.txt`
+      `echo "$(tr '~' '\n' < CMakeLists.txt)" > CMakeLists.txt`
+  end
   def self.build
-    FileUtils.mkdir 'build'
+    Dir.mkdir 'build'
     Dir.chdir 'build' do
       system "cmake \
+             #{CREW_CMAKE_OPTIONS} \
              -DWITH_GCRYPT=ON \
-             -DCMAKE_INSTALL_PREFIX=#{CREW_PREFIX} \
-             -DLIB_INSTALL_DIR=#{CREW_LIB_PREFIX} \
-             -DCMAKE_BUILD_TYPE=Release .."
```

Comment:
You might be able to do something like this instead of the patch:
```
-DCMAKE_C_FLAGS='-fstack-protector-strong'
```

Reply:
Done